src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx

```diff
@@ -17,7 +17,8 @@ function countValidatorsState(t: TFunction, validators: Validator[] | undefined)
   if (!validators) return []
 
   return [
-    { label: t('validator.active'), value: validators.filter(v => v.active).length },
+    { label: t('validator.active'), value: validators.filter(v => v.in_validator_set).length },
+    { label: t('validator.waiting'), value: validators.filter(v => !v.in_validator_set && v.active).length },
```

Comment:
Sync API spec again - current description of `v.active` is the same as `v.in_validator_set`

Reply:
description updated